src/board/UBBoardView.h

```diff
@@ -99,6 +99,10 @@ class UBBoardView : public QGraphicsView
     virtual void mouseMoveEvent(QMouseEvent *event);
     virtual void mouseReleaseEvent(QMouseEvent *event);
     virtual void wheelEvent(QWheelEvent *event);
+
+    int _numScheduledScalings;
```

Comment:
Please use the naming conventions used in Openboard (starting a member variable with `m`) and place it where the other member variables are.